log/logrus/logrus_logger.go

```diff
@@ -12,13 +12,20 @@ import (
 
 type logrusLogger struct {
 	logrus.FieldLogger
+	logrus.Level
 }
 
 var errMissingValue = errors.New("(MISSING)")
 
 // NewLogrusLogger returns a go-kit log.Logger that sends log events to a Logrus logger.
 func NewLogrusLogger(logger logrus.FieldLogger) log.Logger {
-	return &logrusLogger{logger}
+	return &logrusLogger{logger, logrus.InfoLevel}
+}
+
+// NewLogrusLoggerWithLevel returns a go-kit log.Logger that sends log events to a Logrus logger, which
+// will be logged with provided error level
+func NewLogrusLoggerWithLevel(logger logrus.FieldLogger, level logrus.Level) log.Logger {
```

Comment:
For consistency with other Go kit packages we should use the functional option pattern for this.

Reply:
Done, please check :)